src/Collections/ExportableCollection.php

```
@@ -19,30 +22,34 @@
      * @param  string|array  $value
      * @param  string|null  $key
      */
-    public function pluck($value, $key = null): ExportableCollection
+    public function pluck($value, $key = null): SupportCollection
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)